Description: fixes test suite to not expect the year to be between 2000 and 2009.
Origin: upstream, http://svn.apache.org/viewvc?view=rev&rev=954335
Bug: https://issues.apache.org/jira/browse/LOGCXX-365
Forwarded: not-needed
Applied-Upstream: commit 954335
Last-Update: 2014-03-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/test/cpp/util/filter.h	2010/06/14 00:24:42	954334
+++ b/src/test/cpp/util/filter.h	2010/06/14 00:30:43	954335
@@ -31,7 +31,7 @@
 #define BASIC_PAT "\\[0x[0-9A-F]*] (FATAL|ERROR|WARN|INFO|DEBUG)"
 #define ISO8601_PAT "[0-9]\\{4\\}-[0-9]\\{2\\}-[0-9]\\{2\\} [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\},[0-9]\\{3\\}"
 #define ABSOLUTE_DATE_AND_TIME_PAT \
-        "[0-9]\\{1,2\\} .* 200[0-9] [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\},[0-9]\\{3\\}"
+        "[0-9]\\{1,2\\} .* 2[0-9][0-9][0-9] [0-9]\\{2\\}:[0-9]\\{2\\}:[0-9]\\{2\\},[0-9]\\{3\\}"
 #define ABSOLUTE_TIME_PAT "[0-2][0-9]:[0-9][0-9]:[0-9][0-9],[0-9][0-9][0-9]"
 #define RELATIVE_TIME_PAT "^[0-9]+"
 
